nima_core/storage/unified_db.py

```
@@ -323,16 +401,22 @@ def unified_retrieve(
+        try:
+            fts_rows = conn.execute(
+                """
+                SELECT rowid, rank FROM memory_fts
+                WHERE memory_fts MATCH ?
+                ORDER BY rank
+                LIMIT ?
+                """,
+                (fts_query, limit * 3),
+            ).fetchall()
+            fts_scores = {row[0]: row[1] for row in fts_rows}
+        except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)